justbuild.it-ConsoleApp/src/main/java/com/isa/control/SubMenuNavigator.java

```diff
@@ -0,0 +1,38 @@
+package com.isa.control;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+public abstract class SubMenuNavigator {
+
+    void choose(){
+
+        boolean inProgress = true;
+        do{
+            Scanner scanner = new Scanner(System.in);
+            int choice = 0;
+            try{
+                choice = scanner.nextInt();
+                if (choice == 1) {
+                    accept();
+                } else if (choice == 2) {
+                    goBackToMenu();
+                    inProgress = false;
+                } else {
+                    System.out.println("Wpisano błędną cyfrę.");
+                }
+            } catch (InputMismatchException e) {
+                System.out.println("Wpisano niewłaściwe znaki");
+            }
+        } while(inProgress);
+    }
+    void accept(){
```

Comment:
trzeba zmienić nazwę metody bo accept to za mało. Może acceptAction czy cos?

Reply:
zmienione